doc/API.md

```diff
@@ -360,9 +503,236 @@ Function: Submit a header that has passed the POW.
+
+Renter
+------
+
+Queries:
+
+* /renter/downloadqueue     [GET]
+* /renter/load              [POST]
+* /renter/loadascii         [POST]
+* /renter/rename            [POST]
+* /renter/files             [GET]
+* /renter/delete/<path>     [POST]
+* /renter/download/<path>   [GET]
+* /renter/share/<path>      [GET]
+* /renter/shareascii/<path> [GET]
+* /renter/upload/<path>     [POST]
+
+#### /renter/downloadqueue [GET]
+
+Function: Lists all files in the download queue.
+
+Parameters: none
+
+Response:
+```
+struct {
+	downloads []struct {
+		nickname    string
+		destination string
+		filesize    uint64
+		received    uint64
+		starttime   Time (string)
+	}
+}
+```
+'nickname' is the nickname given to the file when it was uploaded.
+
+'destination' is the path that the file was downloaded to.
+
+'filesize' is the size of the file being download.
+
+'received' is the number of bytes downloaded thus far.
+
+'starttime' is the time at which the download was initiated.
+
+#### /renter/files
+
+Function: Lists the status of all files.
+
+Parameters: none
+
+Response:
+```
+struct {
+	files []struct {
+		nickname       string
+		filesize       uint64
+		available      bool
+		uploadprogress float32
+		expiration     types.BlockHeight (uint64)
+	}
+}
+```
+'nickname' is the nickname given to the file when it was uploaded.
+
+'filesize' is the size of the file in bytes.
+
+'available' indicates whether or not the file can be downloaded immediately.
+
+'uploadProgress' is the current upload percentage of the file, including
+redundancy. In general, files will be available for download before
+UploadProgress == 100.
+
+'expiration' is the block height at which the file ceases availability.
+
+#### /renter/load [POST]
+
+Function: Load a .sia file into the renter.
+
+Parameters:
+```
+filename string
+```
+'filename' is the filepath of the .sia file that is being loaded.
+
+Response:
+```
+struct {
+	filesadded []string
+}
+```
+'filesadded' is a list of the nicknames of the files contained in the .sia file
+that was loaded.
+
+
+#### /renter/loadascii [POST]
+
+Function: Load a .sia file into the renter.
+
+Parameters:
+```
+file string
+```
+'file' is the ASCII-encoded .sia file that is being loaded.
+
+Response:
+```
+struct {
+	filesadded []string
+}
+```
+
+#### /renter/rename [POST]
+
+Function: Rename a file. Does not rename any downloads or source files, only
+renames the entry in the renter.
+
+Parameters:
+```
+nickname string
+newname  string
+```
+'nickname' is the current name of the file entry.
+
+'newname' is the new name for the file entry.
+
+Response: standard.
+
+#### /renter/delete/<path> [POST]
+
+Function: Deletes a renter file entry. Does not delete any downloads or
+original files, only the entry in the renter.
+
+Parameters:
+```
+path string
+```
+'path' is the nickname of the file.
```

Comment:
I think we should consider transitioning from "nickname" to "path". We may move path handling to siad in order to facilitate operations like, e.g. deleting a folder. "Nickname" is unintuitive for a newcomer anyway -- "What's a nickname? I just want to drag a file from my desktop into Sia."

Also, worth noting that you won't actually use the name "path" on the API side; you just type `/renter/delete/foo/bar/baz`.